servers/apps/matterbridge/matterbridge.toml

```diff
@@ -0,0 +1,25 @@
+[matrix]
+[matrix.main]
+Server="https://matrix.tchncs.de"
+MxID="@fu-bot:tchncs.de"
+Token="REDACTED"
+
+HTMLDisable=false
+# UseUserName shows the username instead of the server nickname
+UseUserName=false
+KeepQuotedReply=true
+ShowJoinPart=true
+
+[telegram]
+[telegram.main]
+Token="REDACTED"
```

Comment:
is this token associated with an individual user or is there a bot account like the fu-bot matrix account?

Reply:
This is a bot account generated by telegram's BotFather and is tied to a specific telegram user. Its currently linked to my telegram account to be specific.